management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -458,4 +459,27 @@ public void clearSnapshots(@RpcParam(name="snapshotNames") List<String> snapshot
         }
     }
 
+    @Rpc(name = "repair")
+    public void repair(@RpcParam(name="keyspaceName") String keyspace, @RpcParam(name="tables") List<String> tables, @RpcParam(name="full") Boolean full) throws IOException
+    {
+        logger.warn("KEYSPACE = {} ; TABLES = {} ; FULL = {}", keyspace, tables, full);
```

Comment:
Shouldn't this be moved to debug level?

Reply:
good catch, it should probably just be removed.